core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1075,7 +1073,7 @@ public void terminateWorkflow(Workflow workflow, String reason, String failureWo
 			e.printStackTrace();
 		}
 
-		if (((isRequiredInReporting && errorId != 0) || errorId == 0) && !workflow.isSubWorkflow()) {
+		if (!workflow.isSubWorkflow()) {
```

Comment:
is it the same as below? 
`if ((isRequiredInReporting || errorId == 0) && !workflow.isSubWorkflow())`